### DISCUSSION TITLE
How do I restrict search to one Sharepoint page?

### DISCUSSION THREAD

**Discussion by DanaG-pdx** [microsoft-search/pnp-modern-search#4021] [Q&A]
Hello Github Community!

I'm new to the PNP modern search tools, and also not technically skilled. I work in Compliance and I'm trying to create an FAQ resource for staff. Appreciate any help you can provide to help me on my journey :)

I've successfully added the search box and search results webparts to my Sharepoint page.

By default they are setup to provide results from our entire Sharepoint site. However, I’d like to restrict the source to one page (Q&A document library) and cannot figure out how to accomplish this. Are any of you able to help with this?

Thanks,
Dana

**Comment by kasperbolarsen** [comment]
If you intend to surface the FAQ primarily in search, you might want to have a look at out of the box  Microsoft search first, https://learn.microsoft.com/en-us/microsoftsearch/manage-qas

**Comment by DanaG-pdx** [comment]
Hi Kasper, thank you so much for your reply and suggestion. I visited the
website and looked through the instructions. I may be misunderstanding, but
it looks like the source has to be a spreadsheet? If yes, I don't
believe this would work for us. Our source material is all PDF documents
since our "answers" serve as job aids and often include images, hyperlinks,
etc.

**Reply by kasperbolarsen** [reply]
The common definition of a FAQ/Q&A is closer to the setup that the link describes. If your use case includes structured documents as the answer, I would properly call it a custom document search center :-)
I would properly use the Path property to restrict the query to just that library, and I would recommend watching some of these videos in order to get started:
https://www.youtube.com/results?search_query=pnp+modern+search
